docs/README.md

```diff
@@ -1,131 +1,80 @@
 
-#### Data Layer Variables
+It is possible to install the VTEX native GTM app in your store either by using App Store or the VTEX IO Toolbelt.
 
-1. Click in Variable Configuration
-2. Choose "Data Layer Variable"
-3. Type "campaignMedium" in "Data Layer Variable Name"
-4. Click Save, and save as "Data Layer Variable - campaignMedium"
+### Using VTEX App Store
 
-Do the same thing for the variables: "campaignName", "campaignSource" and "userId".
+1. Access the **Apps** section in your account's admin page and look for the Google Tag Manager box; 
+2. Then, click on the **Install** button;
+3. You'll see a warning message about needing to enter the necessary configurations. Scroll down and type in your **GTM ID** in the `Google Tag Manager` field.
+4. Click on **Save**.
 
-#### Google Analytics Variables
+:information_source: Access the [Google Tag Manager page](https://tagmanager.google.com/)</a> and login to you account in order to find out what is your account **GTM ID**. The number your should use is the one provided by the `Container ID` column. 
 
-##### Default
+### Using VTEX IO Toolbelt
 
-1. Click in Variable Configuration
-2. Choose "Google Analytics Settings"
-3. Type your Tracking ID
-4. Click in Ecommerce
-5. Click in "Enable Enhanced Ecommerce Features"
-6. Click in "Use data layer"
-7. Click Save, and save as "Google Analytics" 
+1. [Install](https://vtex.io/docs/recipes/development/installing-an-app/) the `vtex.google-tag-manager@2.x` app. You can confirm that the app has now been installed by running `vtex ls` again. 
+2. Access the **Apps** section in your account's admin page and look for the Google Tag Manager box. Once you find it, click on the box.
+3. Fill in the `Google Tag Manager` field with your **GTM ID**. 
+4. Click on **Save**.
 
-If you intend to use the [User ID feature of Google Analytics](https://support.google.com/analytics/answer/3123662), you need to set a field using the userId variable previously created:
+:information_source: Access the [Google Tag Manager page](https://tagmanager.google.com/)</a> and login to you account in order to find out what is your account **GTM ID**. The number your should use is the one provided by the `Container ID` column.
 
-1. Click in Fields to Set
-2. Add the field:
-Field Name | Value
----|---
-userId | {{userId}}
+After installing the app, you are ready to use your GTM as usual by accessing your account dashboard directly on the [Google Tag Manager](https://tagmanager.google.com/) page. 
 
-##### Checkout
+## Restrictions
 
-1. Click in Variable Configuration
-2. Choose "Google Analytics Settings"
-3. Type your Tracking ID
-4. Click in More Settings
-5. Click in Fields to Set
-6. Add the fields:
+In order to avoid performance problems and unforeseen behavior, our VTEX IO Google Tag Manager solution uses the native GTM **blocklist** feature. You can read more about this feature on the [Google Developer Guide](https://developers.google.com/tag-manager/devguide).
```

Comment:
```suggestion
In order to avoid performance problems and unforeseen behavior, our VTEX IO Google Tag Manager solution uses the native GTM **blacklist** feature. You can read more about this feature on the [Google Developer Guide](https://developers.google.com/tag-manager/web/restrict).
```